.github/workflows/deploy-component-library.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       - name: Setup Node
         uses: actions/setup-node@v2
         with:
-          node-version: '16.10'
+          node-version: '16.13'
```

Comment:
Do we want to bump this to `16.13.1` since that's the latest LTS?

Reply:
`16.13` will install the latest bug-fix version, so if `16.13.1` is the latest, that'll get installed. And then when `16.13.2` is released, this version string will install that.